event_routing_backends/models.py

```diff
@@ -40,6 +43,80 @@ def get_value_from_dotted_path(dict_obj, dotted_key):
     return result
 
 
+class RouterConfigurationModelManager(models.Manager):
+    """
+    Query manager for ConfigurationModel
+    """
+    def _current_ids_subquery(self):
```

Comment:
I don't think we need these methods
`_current_ids_subquery `, `current_set`, `with_active_flag `

Reply:
Done